DQM/SiPixelPhase1Common/src/SiPixelCoordinates.cc

```diff
@@ -0,0 +1,916 @@
+// -*- C++ -*-
```

Comment:
My proposal is to move it to DataFormats/​SiPixelDetId.
I think this is what David had in mind.